content/courses/tokens-and-nfts/token-program.md

```diff
@@ -464,7 +467,7 @@ Assuming you already have a `.env` file with a `SECRET_KEY` setup per
 [Cryptography fundamentals](/content/courses/intro-to-solana/intro-to-cryptography.md).
 
 ```bash
-npm i @solana/web3.js @solana/spl-token @solana-developers/helpers esrun
+npm i @solana/web3.js @solana/spl-token @solana-developers/helpers@2.2.0 esrun
```

Comment:
We did have some issues, but they should be fixed now. No need to ask people to install a specific version.